org.eclipse.wb.swing/.classpath

```diff
@@ -1,7 +1,7 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<classpath>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/J2SE-1.5"/>
-	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
-	<classpathentry kind="src" path="src"/>
-	<classpathentry kind="output" path="bin"/>
-</classpath>
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
This looks unrelated, please update wb.swing to Java 11 (via the MANIFEST.MF) and update the classpath

Please send a new PR for this and remove this change from this PR.

![image](https://user-images.githubusercontent.com/139910/166448111-ed68953e-940b-469c-af00-1b251966c849.png)

Reply:
@vogella I've made the changes you requested

Reply:
@vogella  requested changes applied